tests/nftables/test_nftables.py

```diff
@@ -0,0 +1,191 @@
+import ipaddress
+import subprocess
+import unittest
+import json
+import os
+import sys
+from time import sleep
+from typing import List
+from ipaddress import ip_address
+
+from tests.nftables import mock_lapi
+
+SCRIPT_DIR = os.path.dirname(os.path.realpath(__file__))
+os.chdir(SCRIPT_DIR)
```

Comment:
These global-level function calls are run when you import a module, possibly causing side effects. Better to change directory in the explicit control flow (i.e. main() or whatever is the entry point). May not be a problem with two files, but when you have multiple os.chdir() of this kind, import order can trigger bugs